src/components/gig/card.tsx

```diff
@@ -141,7 +149,10 @@ const GigCard = async ({
                   checkbox1={gig.checkbox1}
                   checkbox2={gig.checkbox2}
                   signup={signup ?? undefined}
-                  signupEndExsists={!!gig.signupEnd}
+                  signupOptions={SIGNUP_OPTIONS.slice(
+                    0,
+                    allowMaybe ? SIGNUP_OPTIONS.length : -1,
+                  )}
```

Comment:
I feel we should explicitly define the two possibilities here instead of using a slice, as it would be clearer for future readers. How about defining `SIGNUP_OPTIONS = { yes: ..., no: ..., maybe: ... }` and using them here?